src/test/java/com/imsweb/geocoder/GeocodeOutputTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Information Management Services, Inc.
+ */
+package com.imsweb.geocoder;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import junitx.util.PrivateAccessor;
+
+public class GeocodeOutputTest {
+
+    // The important behavior in the GeocodeOutput class is tested in GeocoderTest, but wanted a test for some of the helper methods
+    @Test
+    public void testIntValue() throws Throwable {
+        Class<?>[] clazz = {String.class};
+
+        Object[] param = {"7"};
+        Object result = PrivateAccessor.invoke(GeocodeOutput.class, "intValue", clazz, param);
```

Comment:
I agree, actually, about using the PrivateAccessor, but I was reluctant to change access modifier. I'll be less reluctant in the future!